Terminal.Gui/Configuration/ConfigurationManager.cs

```diff
@@ -235,6 +235,14 @@ public bool Apply ()
 		[SerializableConfigurationProperty (Scope = typeof (SettingsScope), OmitClassName = true),
 			JsonPropertyName ("Glyphs")]
 		public static GlyphDefinitions Glyphs { get; set; } = new GlyphDefinitions ();
+
+		/// <summary>
+		/// If <see langword="true"/>, enables the use of Nerd unicode symbols.
+		/// This requires specific font(s) to be installed on the users machine
+		/// to work correctly.  Defaults to <see langword="false"/>.
+		/// </summary>
+		[SerializableConfigurationProperty (Scope = typeof (SettingsScope), OmitClassName = true)]
+		public static bool UseNerd { get; set; } = false;
```

Comment:
I didn't want CM to fill up with setting definitions. I put Glyphs here because i felt neither `Application` or `ConsoleDriver` was ideal.

Maybe making them ThemeScope properties makes more sense?

Reply:
I wasn't sure where it should go either! But yeah theme sounds like a sensible place for it. I will move it there when I get a chance.